ledger/accountdb.go

```diff
@@ -2685,7 +2686,9 @@ func (qs *accountsDbQueries) lookupKeyValue(key string) (pv persistedKVData, err
 func (qs *accountsDbQueries) lookupKeysByPrefix(prefix string, maxKeyNum uint64, results map[string]bool, resultCount uint64) (round basics.Round, err error) {
 	err = db.Retry(func() error {
 		// Cast to []byte to avoid interpretation as character string, see note in upsertKvPair
-		rows, err := qs.lookupKeysByPrefixStmt.Query(len(prefix), []byte(prefix))
+		prefixIncrBigInt := new(big.Int).Add(new(big.Int).SetBytes([]byte(prefix)), big.NewInt(1))
+		prefixIncrBytes := prefixIncrBigInt.Bytes()
```

Comment:
It feels a little funny to use big.Int here.  I guess I don't hate it, but doing the little `for` loop that walks up the byte slice seems much faster. (And I noticed you benchmark had to do 6 more allocations this way.)

Reply:
yea, this is just for a demo-ish use, I would like to trigger the DB optimization and benchmark it first.

For down-the-line work, I will separately make a small function to take care of increment and corner case as we discussed. And a test case for that.

Reply:
Add a function to handle the corner case, and together with an additional sql query statement, and a test case.